src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLanguageServerClient.cs

```diff
@@ -111,7 +109,7 @@ internal class RazorLanguageServerClient : ILanguageClient, ILanguageClientCusto
             _server = await RazorLanguageServer.CreateAsync(serverStream, serverStream, traceLevel, ConfigureLanguageServer).ConfigureAwait(false);
 
             // Fire and forget for Initialized. Need to allow the LSP infrastructure to run in order to actually Initialize.
-            _server.InitializedAsync(token).FileAndForget("RazorLanguageServerClient_ActivateAsync");
```

Comment:
Does `FileAndForget` to any logging of exceptions or anything that might be useful?

Reply:
Looks like for exceptions it'll log that information into the ActivityLog and then it also looks like it'll run bits of it on the joinable task factory to prevent UI deadlocks? Digging through the code there's a lot that happens in the consumption of the joinable task. Given our server acts truly OOP deadlocks wont be possible there (it doesn't even know about the UI Thread) so this might actually be a net-win gvien we've also never seen any logs in the ActivityLog related to this!